src/Combinatorics/Matroids/realization_space.jl

```diff
@@ -0,0 +1,989 @@
+
+
+mutable struct MatroidRealizationSpace
+    defining_ideal::Union{Ideal, NumFieldOrdIdl, Nothing}
+    inequations::Union{Vector{Oscar.RingElem},Nothing}
```

Comment:
You could just as well have this as
```suggestion
    inequations::Vector{Oscar.RingElem}
```
and then in the code use `isdefined(obj,:inequations)` instead of `!isnothing(obj.inequations)`. Same for other fields.

Reply:
Fixed.